### PR TITLE
feat(loader): add scale property

### PR DESCRIPTION
**Pull request by asangma**
Not sure how we want to handle scale with Loader.
I started down a path, but yeah...would like any suggestions.

### CODE REVIEW

**Comment by bstifle** [comment]
image
@asangma

**Comment by paulcpederson** [comment]
@asangma Yeah I think you are going down the right path, thanks for tackling this. Giving inline loaders their own scale is the best bet. This will be useful when using loaders inside buttons because we can use the corresponding scale of the button (sml). 

Probably just extending what you've done here and adding an additional object like `inlineSizes` with the dimensions above. Would be great to update the button component to leverage the scale prop once inline scale is added as well!

**Comment by asangma** [comment]
Thanks @bstifle! I was about to hit you up for that!

Thanks for the feedback @paulcpederson!

**Comment by asangma** [comment]
@bstifle Do you have stroke widths for each of those scales?

**Comment by bstifle** [comment]
@asangma 

image

**Comment by asangma** [comment]
Aight y'allz. I think this is ready for review.
@bstifle Might be some spacing stuff to look at between the loader and loader-text.

**Comment by asangma** [comment]
@bstifle Also, I made some choices re: font-sizes. So would apreesh feedbarks.

File: src/components/calcite-loader/calcite-loader.tsx [hunk 1]
```diff
@@ -20,9 +20,11 @@ export class CalciteLoader {
   //
   //--------------------------------------------------------------------------
   /** Show the loader */
-  @Prop({ reflect: true }) isActive: boolean = false;
+  @Prop({ reflect: true }) active: boolean = false;
   /** Inline loaders are smaller and will appear to the left of the text */
   @Prop({ reflect: true }) inline: boolean = false;
+  /** Speficy the scale of the loader. Defaults to "m" */
+  @Prop({ reflect: true }) scale: "s" | "m" | "l" = "m";
```
Comment (asangma) [inline src/components/calcite-loader/calcite-loader.tsx:27]: @macandcheese @paulcpederson I think we had an example of a check somewhere that validated this prop and set it to the default if a rando value were passed yeah?

File: src/components/calcite-loader/calcite-loader.tsx [hunk 2]
```diff
@@ -38,20 +40,23 @@ export class CalciteLoader {
   //
   //--------------------------------------------------------------------------
   render() {
-    const id = this.el.id || this.guid;
-    const size = this.inline ? 20 : 56;
-    const radius = this.inline ? 9 : 25;
+    const { el, inline, scale, text, type, value } = this;
+
+    const id = el.id || guid;
```
Comment (macandcheese) [inline src/components/calcite-loader/calcite-loader.tsx:45]: Do we need to use this at all here?

Comment (asangma) [reply src/components/calcite-loader/calcite-loader.tsx:45]: I didn't see it being used. I can remove it.

**Comment by asangma** [comment]
@macandcheese FYI. Adding a couple e2es.

**Comment by asangma** [comment]
E2E updated.

**Comment by bstifle** [comment]
Perfect!!!! nice work

**Comment by asangma** [comment]
Thanks @bstifle!

**Pull request merged**